<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compTooltip</class>
 <widget class="QWidget" name="compTooltip">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Component details</string>
  </property>
  <widget class="QLabel" name="l1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean</string>
   </property>
  </widget>
  <widget class="QLabel" name="l1_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Covariance</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="inversion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>90</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inverted</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="outlier">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Outlier</string>
   </property>
  </widget>
  <widget class="QLabel" name="l1_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Elements</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="obsolete">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Obsolete</string>
   </property>
  </widget>
  <widget class="QLabel" name="l1_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Redirected to</string>
   </property>
  </widget>
  <widget class="QLabel" name="meanX">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>5</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="meanY">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>5</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="covX">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>35</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="corrXY">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>35</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="corrYX">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>60</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="covY">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>60</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="elements">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>115</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="redirectedTo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>225</y>
     <width>366</width>
     <height>46</height>
    </rect>
   </property>
   <property name="title">
    <string>Decay</string>
   </property>
   <widget class="QCheckBox" name="decayEnabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1_6">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>20</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Counter</string>
    </property>
   </widget>
   <widget class="QLabel" name="decayCounter">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="decayCurrent">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
  </widget>
  <widget class="QListView" name="traceList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>356</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l1_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Trace</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="blocked">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Blocked</string>
   </property>
  </widget>
  <widget class="QLabel" name="l1_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>165</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Parent</string>
   </property>
  </widget>
  <widget class="QLabel" name="parent">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>165</y>
     <width>256</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="sourceNode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>256</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="l1_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Source node</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
